projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -112,10 +112,13 @@ public function render_initial_state() {
 	 * @return array
 	 */
 	public function initial_state() {
+		global $publicize;
+
 		return array(
 			'apiRoot'           => esc_url_raw( rest_url() ),
 			'apiNonce'          => wp_create_nonce( 'wp_rest' ),
 			'registrationNonce' => wp_create_nonce( 'jetpack-registration-nonce' ),
+			'connections'       => $publicize->get_all_connections_for_user(),
```

Comment:
I don't know, but we might want to think about caching here - definitely not right now though!

I think this could trigger a refresh of the connections and that would delay the rendering of the page. In future we could ensure the refresh only happens as an async request once the page is loaded, or something similar.

Reply:
Got you! Will keep that in mind.